includes/class-wc-webhook.php

```diff
@@ -50,6 +50,7 @@ class WC_Webhook extends WC_Legacy_Webhook {
 		'user_id'          => 0,
 		'api_version'      => 3,
 		'pending_delivery' => false,
+		'retry_enabled'    => false,
```

Comment:
Just wanted to note that any new metadata will also need to be added to the `WC_Webhook_Data_Store` class!

Reply:
I was looking into making this change, it seems edits to that class (could/should) affect the database structure.  Is there a way of proposing schema changes?  Or should I just make the changes to the `class-wc-install.php` file?  (There must be a way for migrations to be handled, or should I just skip the db for now?)

Reply:
We use WordPress' [dbDelta function](https://developer.wordpress.org/reference/functions/dbdelta/) to handle database migrations. If you change the schema and update the database version it _should_ automatically execute `ALTER TABLE` statements to match the new schema.

Reply:
After looking at the code I'm still not sure I completely understand.  I made the changes and got my table updated by deactivating/activating the plugin.  It seems I might need to create a function in `wc-updated-functions.php`, but I didn't want to preemptively create a new woocommerce version. (Or should this be a semver change for woo -- 4.5.0 or 4.4.1 -- not sure what the `@since` should be in the php docs either).